version.props

```diff
@@ -1,13 +1,20 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>2.0.7</VersionPrefix>
-    <VersionSuffix>rtm</VersionSuffix>
+    <AspNetCoreMajorVersion>2</AspNetCoreMajorVersion>
+    <AspNetCoreMinorVersion>0</AspNetCoreMinorVersion>
+    <AspNetCorePatchVersion>7</AspNetCorePatchVersion>
```

Comment:
Does this track the .NET Core version too? Reason is that for the hosting bundle, the branding is always based off the runtime version we redist and not any ASP.NET versions

Reply:
For 2.x, these should always align. We have to re-release an aspnetcore runtime installer for each .NET Core release, and we are planning to always align them